crates/runtime-datafusion-udfs/src/bucket.rs

```diff
@@ -206,7 +207,8 @@ mod tests {
                 ColumnarValue::Scalar(ScalarValue::Utf8(Some("test".to_string()))),
             ],
             number_rows: 1,
-            return_type: &DataType::Int32,
+            arg_fields: vec![],
+            return_field: Arc::new(Field::new("ignored_name", DataType::Int32, false))
```

Comment:
I was trying to get the tests to build and this was the next to-do. Maybe want to infer nullability from the args?